aktualizr.rb

```diff
@@ -1,32 +1,30 @@
 class Aktualizr < Formula
   desc "C++ Client for HERE OTA Connect"
   homepage "https://github.com/advancedtelematic/aktualizr.git"
-
-  head "https://github.com/advancedtelematic/aktualizr.git"
-
   version = "2020.3"
 
-  url "https://github.com/advancedtelematic/aktualizr.git", :using => :git, :tag => "#{version}"
+  url "https://github.com/advancedtelematic/aktualizr.git", :using => :git, :tag => "#{version}", :revision => "9f2cc5d7b026eb1a1404f0c77ef6cb9377245423"
```

Comment:
Looks like the revision is the only real functional change here. Seems fine, though.

Reply:
not exactly that tool is also sensitive to an order of dependencies for some reason.